bika/lims/browser/instrument.py

```diff
@@ -532,9 +532,10 @@ def folderitems(self):
                     error_amount = ((target / 100) * error) if target > 0 else 0
                     upper = smax + error_amount
                     lower = smin - error_amount
-
+                    cap_date = obj.getResultCaptureDate()
+                    cap_date = cap_date and cap_date.strftime('%Y-%m-%d %I:%M %p') or ''
```

Comment:
maybe better use `isinstance(cap_date, DateTime)` or add a `is_date` function to the API. I've seen already cases where we got weird stuff somehow...

Reply:
Yes, you are right, will add this change. Thanks!